<appName> only allow alphanumeric characters

When trying to deploy the `javase-quarkus` sample with the Azure Maven plugin, I get an error that the `<appName> only allow alphanumeric characters`. That's because, in this example, the `example.postfix` variable is not set and you end up with a wrong app name:

```
<appName>javase-quarkus-${example.postfix}</appName>
```

To fix it it's just a matter of adding the variable:

```
<example.postfix>${maven.build.timestamp}</example.postfix>
<maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
<appName>javase-quarkus-${example.postfix}</appName>
```

The variable is actually defined in `spring-cloud/helloworld/pom.xml` but maybe should be defined in the parent pom.

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce

Once you've packaged the Quarkus application with `mvn package` you follow the readme and deploy the app with:

```
 mvn azure-webapp:deploy \
                                               -DappName=$APPSERVICE_JAVASE_QUARKUS \
                                               -DappServicePlan=$APPSERVICE_PLAN \
                                               -DresourceGroup=$RESOURCE_GROUP
```

Then you get the error.

### Any log messages given by the failure

```
[ERROR] Failed to execute goal com.microsoft.azure:azure-webapp-maven-plugin:1.12.0:deploy (default-cli) on project appservice-javase-quarkus: The <appName> only allow alphanumeric characters, hyphens and cannot start or end in a hyphen.
```

--- a/appservice/deploy-to-deployment-slot/pom.xml
+++ b/appservice/deploy-to-deployment-slot/pom.xml
@@ -53,7 +53,7 @@
         </dependency>
     </dependencies>
     <properties>
-        <appName>helloworld-${postfix}</appName>
+        <appName>helloworld-${example.postfix}</appName>
         <appServicePlan>azure-examples-appserviceplan</appServicePlan>
         <deploymentSlotName>staging</deploymentSlotName>
         <javaVersion>jre8</javaVersion>

--- a/appservice/jboss-eap-helloworld/pom.xml
+++ b/appservice/jboss-eap-helloworld/pom.xml
@@ -50,7 +50,7 @@
         </dependency>
     </dependencies>
     <properties>
-        <appName>helloworld-jboss-eap-${postfix}</appName>
+        <appName>helloworld-jboss-eap-${example.postfix}</appName>
         <appServicePlan>azure-examples-appserviceplan</appServicePlan>
         <javaVersion>jre8</javaVersion>
         <region>westus2</region>

--- a/appservice/tomcat-helloworld/pom.xml
+++ b/appservice/tomcat-helloworld/pom.xml
@@ -50,7 +50,7 @@
         </dependency>
     </dependencies>
     <properties>
-        <appName>helloworld-${postfix}</appName>
+        <appName>helloworld-${example.postfix}</appName>
         <appServicePlan>azure-examples-appserviceplan</appServicePlan>
         <javaVersion>jre8</javaVersion>
         <region>westus2</region>

--- a/pom.xml
+++ b/pom.xml
@@ -90,6 +90,9 @@
         </profile>
     </profiles>
     <properties>
+        <example.prefix>examples</example.prefix>
+        <example.postfix>${maven.build.timestamp}</example.postfix>
+        <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 </project>

--- a/spring-cloud/helloworld/pom.xml
+++ b/spring-cloud/helloworld/pom.xml
@@ -16,11 +16,8 @@
     
     <properties>
         <example.appName>${example.prefix}-springcloud-helloworld</example.appName>
-        <example.postfix>${maven.build.timestamp}</example.postfix>
-        <example.prefix>examples</example.prefix>
         <example.resourceGroup>${example.prefix}-rg</example.resourceGroup>
         <example.springCloudName>${example.prefix}-springcloud</example.springCloudName>
-        <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
         <java.version>1.8</java.version>
     </properties>
 

